tests/test-vip-mail.php

```diff
@@ -1,9 +1,17 @@
 <?php
 
+
 class VIP_Mail_Test extends \WP_UnitTestCase {
 	public function setUp() {
 		parent::setUp();
 		reset_phpmailer_instance();
+		if ( ! defined( 'USE_VIP_PHPMAILER' ) ) {
+			define( 'USE_VIP_PHPMAILER', true );
+		}
+	}
+
+	public function tearDown(): void {
+		reset_phpmailer_instance();
```

Comment:
Why run this at the end of each test, when it's also run at the start of each test?

Reply:
I tried removing it and it seemed fine for the specific test file. I pushed the change to run all the test pipelines.

Reply:
@GaryJones I mostly wanted to make sure the global state is not polluted AFTER the tests are done, is that not the right way to do it?